VideoLocker/res/layout/discussion_responses_thread_row.xml

```diff
@@ -37,6 +37,15 @@
             android:textSize="@dimen/edx_small"
             tools:text="Thread body" />
 
+        <TextView
+            android:id="@+id/discussion_responses_thread_row_visibility_text_view"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/discussion_responses_standard_margin"
```

Comment:
@aleffert: Here you go:

With Margin | Without Margin
------------|---------------
![cohort_visibility_with_margin](https://cloud.githubusercontent.com/assets/4244109/12102195/046a03e8-b35b-11e5-8804-82eb38cd256d.png) | ![cohort_visibility_without_margin](https://cloud.githubusercontent.com/assets/4244109/12102202/11ddcdc0-b35b-11e5-94fd-e46f2c3a8ab2.png)